app/src/main/res/layout/activity_closet.xml

```diff
@@ -1,82 +1,375 @@
-</android.support.constraint.ConstraintLayout>
\ No newline at end of file
+        android:background="#F0F0F0">
+
+        <TextView
+            android:id="@+id/closetTitle"
+            android:layout_width="0dp"
+            android:layout_height="60dp"
+            android:layout_marginStart="5dp"
+            android:layout_marginTop="10dp"
+            android:layout_marginEnd="10dp"
+            android:text="@string/closet"
+            android:textAlignment="center"
+            android:textSize="40sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/closetBackButton"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <Button
+            android:id="@+id/closetBackButton"
+            android:layout_width="wrap_content"
+            android:layout_height="60dp"
+            android:layout_marginStart="10dp"
+            android:layout_marginTop="10dp"
+            android:onClick="backToMainScreen"
+            android:text="@string/back"
+            android:textSize="20sp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/shirtsHeader"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="10dp"
+            android:layout_marginTop="10dp"
+            android:layout_marginEnd="10dp"
+            android:text="@string/shirts"
+            android:textSize="30sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/closetBackButton" />
+
+
+        <HorizontalScrollView
+            android:id="@+id/shirtScrollView"
+            android:layout_width="0dp"
+            android:layout_height="150dp"
+            android:layout_marginStart="10dp"
+            android:layout_marginTop="5dp"
+            android:layout_marginEnd="10dp"
+            android:background="#FFFFFF"
+            android:visibility="visible"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/shirtsHeader">
+
+
+            <LinearLayout
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:orientation="horizontal">
+
+                <ImageView
+                    android:id="@+id/shirt1"
+                    android:layout_width="150dp"
+                    android:layout_height="150dp"
+                    android:layout_marginStart="5dp"
+                    android:layout_marginEnd="5dp"
+                    app:srcCompat="@drawable/shirt"
+                    android:contentDescription="@string/todo" />
```

Comment:
Are we going to have to add an entry in here every time we want to add something to the horizontal scrollbars of each type of clothes. There should be functions in the java code for setting these images automatically with an array of images or something. Just want to make sure this is going to be possible with your current design.

Reply:
Yes, I know that the images I put into the closet sections are temporary, I just haven't gotten around to creating the method yet because I am still working on the set clothing activity. I think being able to implement this design while also using code from java is theoretically possible because I know I can add java objects of the Views, Buttons, etc. and change their information through the findViewById method so I'm confident I can figure out a way to achieve the design.